### PR TITLE
[MH2018-10] Feature/featured categories shopify

### PR DESCRIPTION
**Pull request by rubiocanino**
It's ready to review the featured-categories

Just want some feedback , thank you!

[MH2018-10]
How looks
image

Heroku deployment
image

Original
image

### CODE REVIEW

File: Lil-Peep/src/styles/tools/mixins.scss [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+  }
+
+  .#{$breakpoint}hide {
+    display: none !important;
+  }
+}
+
+
+/*================ Responsive Text Alignment Helper ================*/
+@mixin responsive-text-align-helper($breakpoint: '') {
```
Comment (magmahound) [inline Lil-Peep/src/styles/tools/mixins.scss]: Prefer double-quoted strings

File: Lil-Peep/src/styles/tools/mixins.scss [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    }
+  }
+
+  @if $breakpoint-found == false {
+    @warn 'Breakpoint "#{$media-query}" does not exist';
+  }
+}
+
+/*================ Responsive Show/Hide Helper ================*/
+@mixin responsive-display-helper($breakpoint: '') {
```
Comment (magmahound) [inline Lil-Peep/src/styles/tools/mixins.scss]: Prefer double-quoted strings

File: Lil-Peep/src/styles/tools/mixins.scss [hunk 3]
```diff
@@ -0,0 +1,119 @@
+/*================ Mixins ================*/
+@mixin clearfix() {
+  &::after {
+    content: '';
```
Comment (magmahound) [inline Lil-Peep/src/styles/tools/mixins.scss]: Properties should be ordered clear, content, display<br>Prefer double-quoted strings

File: Lil-Peep/src/styles/tools/mixins.scss [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+  height: 1px;
+  width: 1px;
+  margin: -1px;
+  padding: 0;
+  border: 0;
+}
+
+@mixin visually-shown($position: inherit) {
+  // sass-lint:disable no-important
+  position: $position !important;
```
Comment (magmahound) [inline Lil-Peep/src/styles/tools/mixins.scss]: !important should not be used

File: Lil-Peep/src/styles/tools/mixins.scss [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+
+/*================ Responsive Show/Hide Helper ================*/
+@mixin responsive-display-helper($breakpoint: '') {
+  // sass-lint:disable no-important
+  .#{$breakpoint}show {
+    display: block !important;
+  }
+
+  .#{$breakpoint}hide {
+    display: none !important;
```
Comment (magmahound) [inline Lil-Peep/src/styles/tools/mixins.scss]: !important should not be used

File: Lil-Peep/src/styles/modules/featured-categories.scss [hunk 6]
```diff
@@ -0,0 +1,206 @@
...
+
+    .banner_plush_toys {
+      background-position: 50% 30%;
+      background-repeat: no-repeat;
+      background-size: 100%;
+      width: 50%;
+
+     .plush_toys_content {
+        align-items: flex-end;
+        background: linear-gradient(to bottom,rgba(0, 0, 0, 0) 29%, rgba(0, 0, 0, 0) 49%, rgba(0, 0, 0, 0) 52%, rgba(0, 0, 0, 0.65) 100%);
```
Comment (magmahound) [inline Lil-Peep/src/styles/modules/featured-categories.scss]: Commas in function arguments should be followed by a single space

File: Lil-Peep/src/styles/modules/featured-categories.scss [hunk 7]
```diff
@@ -0,0 +1,206 @@
...
+
+
+        .title {
+          color: white;
+          margin-bottom: 5px;
+          margin-left: 8px;
+        }
+      }
+    }
+    .banner_outdoors {
```
Comment (magmahound) [inline Lil-Peep/src/styles/modules/featured-categories.scss]: Selector `banner_outdoors` should be written in lowercase with hyphens

File: Lil-Peep/src/styles/modules/featured-categories.scss [hunk 8]
```diff
@@ -0,0 +1,206 @@
...
+
+        .title {
+          color: white;
+          margin-bottom: 5px;
+          margin-left: 8px;
+        }
+      }
+    }
+
+    .banner_plush_toys {
```
Comment (magmahound) [inline Lil-Peep/src/styles/modules/featured-categories.scss]: Selector `banner_plush_toys` should be written in lowercase with hyphens

File: Lil-Peep/src/styles/modules/featured-categories.scss [hunk 9]
```diff
@@ -0,0 +1,206 @@
...
+    border-bottom: 1px solid $color_bd;
+    font-family: "Oswald", sans-serif;
+    display: flex;
+    justify-content: center;
+    margin-top: 30px;
+
+    .cont {
+      text-decoration: none;
+    }
+    .banner_wooden_toys {
```
Comment (magmahound) [inline Lil-Peep/src/styles/modules/featured-categories.scss]: Selector `banner_wooden_toys` should be written in lowercase with hyphens

File: Lil-Peep/src/styles/modules/featured-categories.scss [hunk 10]
```diff
@@ -0,0 +1,206 @@
...
+        align-self: center;
+        height: 15px;
+        width: 28px;
+      }
+    }
+  }
+}
+
+@media only screen and (min-width: 769px) {
+  .categories_main {
```
Comment (magmahound) [inline Lil-Peep/src/styles/modules/featured-categories.scss]: Selector `categories_main` should be written in lowercase with hyphens

File: Lil-Peep/src/styles/modules/featured-categories.scss [hunk 11]
```diff
@@ -0,0 +1,206 @@
...
+    }
+
+  }
+
+  .banner_outdoors {
+    background-position: 70% 30%;
+    background-repeat: no-repeat;
+    background-size: cover;
+
+    .outdoors_content {
```
Comment (magmahound) [inline Lil-Peep/src/styles/modules/featured-categories.scss]: Selector `outdoors_content` should be written in lowercase with hyphens

File: Lil-Peep/src/styles/modules/featured-categories.scss [hunk 12]
```diff
@@ -0,0 +1,206 @@
...
+        color: white;
+        margin-bottom: 5px;
+        margin-left: 8px;
+        text-decoration: none;
+      }
+    }
+
+  }
+
+  .banner_outdoors {
```
Comment (magmahound) [inline Lil-Peep/src/styles/modules/featured-categories.scss]: Selector `banner_outdoors` should be written in lowercase with hyphens

File: Lil-Peep/src/styles/modules/featured-categories.scss [hunk 13]
```diff
@@ -0,0 +1,206 @@
...
+
+        .title {
+          color: white;
+          margin-bottom: 5px;
+          margin-left: 8px;
+        }
+      }
+    }
+
+    .banner_plush_toys {
```
Comment (magmahound) [inline Lil-Peep/src/styles/modules/featured-categories.scss]: Selector `banner_plush_toys` should be written in lowercase with hyphens

File: Lil-Peep/src/styles/modules/featured-categories.scss [hunk 14]
```diff
@@ -0,0 +1,206 @@
...
+  .categories_main {
+    border-color: blue;
+    border-style: solid;
+    border: 5px;
+    background-color: $color_bg;
+    border-bottom: 1px solid $color_bd;
+    font-family: "Oswald", sans-serif;
+    margin-top: 15px;
+
+    .banner_wooden_toys {
```
Comment (magmahound) [inline Lil-Peep/src/styles/modules/featured-categories.scss]: Selector `banner_wooden_toys` should be written in lowercase with hyphens

File: Lil-Peep/src/styles/modules/featured-categories.scss [hunk 15]
```diff
@@ -0,0 +1,206 @@
+@media only screen and (max-width: 768px) {
+  .categories_main {
```
Comment (magmahound) [inline Lil-Peep/src/styles/modules/featured-categories.scss]: Selector `categories_main` should be written in lowercase with hyphens

File: Lil-Peep/src/styles/modules/featured-categories.scss [hunk 16]
```diff
@@ -0,0 +1,206 @@
...
+    border-color: blue;
```
Comment (jaimelr) [inline Lil-Peep/src/styles/modules/featured-categories.scss]: This should be a variable defined in `variables.scss`

File: Lil-Peep/src/styles/modules/featured-categories.scss [hunk 17]
```diff
@@ -0,0 +1,206 @@
...
+    border-style: solid;
+    border: 5px;
+    background-color: $color_bg;
+    border-bottom: 1px solid $color_bd;
+    font-family: "Oswald", sans-serif;
```
Comment (jaimelr) [inline Lil-Peep/src/styles/modules/featured-categories.scss]: This should be a variable defined in `variables.scss`

File: Lil-Peep/src/styles/modules/featured-categories.scss [hunk 18]
```diff
@@ -0,0 +1,206 @@
...
+        display: flex;
+        height: 160px;
+        width: 100%;
+
+        .cont {
+          text-decoration: none;
+        }
+
+        .title {
+          color: white;
```
Comment (jaimelr) [inline Lil-Peep/src/styles/modules/featured-categories.scss]: This should be a variable defined in `variables.scss`

File: Lil-Peep/src/styles/modules/featured-categories.scss (hunk 13 above)
Comment (jaimelr) [reply Lil-Peep/src/styles/modules/featured-categories.scss]: Important! :point_up:

File: Lil-Peep/src/styles/modules/featured-categories.scss (hunk 12 above)
Comment (jaimelr) [reply Lil-Peep/src/styles/modules/featured-categories.scss]: Important! :point_up:

File: Lil-Peep/src/styles/modules/featured-categories.scss (hunk 11 above)
Comment (jaimelr) [reply Lil-Peep/src/styles/modules/featured-categories.scss]: Important! :point_up:

File: Lil-Peep/src/styles/modules/featured-categories.scss [hunk 19]
```diff
@@ -0,0 +1,206 @@
...
+    justify-content: center;
+    margin-bottom: 20px;
+    margin-top: 20px;
+    text-align: center;
+
+    .scroll-up-button {
+      background-color: $color_bg_scroll;
+      border-radius: 3px;
+      border-style: none;
+      color: white;
```
Comment (jaimelr) [inline Lil-Peep/src/styles/modules/featured-categories.scss]: This should be a variable defined in `variables.scss`

File: Lil-Peep/src/styles/modules/featured-categories.scss [hunk 20]
```diff
@@ -0,0 +1,206 @@
...
+      display: flex;
+      display: inline-flex;
+      height: 40px;
+      justify-content: center;
+      width: 160px;
+      text-decoration: none;
+
+      .title {
+        font-family: "Oswald", sans-serif;
```
Comment (jaimelr) [inline Lil-Peep/src/styles/modules/featured-categories.scss]: This should be a variable defined in `variables.scss`

File: Lil-Peep/src/styles/modules/featured-categories.scss [hunk 21]
```diff
@@ -0,0 +1,206 @@
...
+        display: flex;
+        align-self: center;
+        height: 15px;
+        width: 28px;
+      }
+    }
+  }
+}
+
+@media only screen and (min-width: 769px) {
```
Comment (jaimelr) [inline Lil-Peep/src/styles/modules/featured-categories.scss]: This should be a variable defined in `variables.scss`

File: Lil-Peep/src/styles/modules/featured-categories.scss [hunk 22]
```diff
@@ -0,0 +1,206 @@
...
+  .categories_main {
+    border-color: blue;
```
Comment (jaimelr) [inline Lil-Peep/src/styles/modules/featured-categories.scss]: This should be a variable defined in `variables.scss`

File: Lil-Peep/src/styles/modules/featured-categories.scss [hunk 23]
```diff
@@ -0,0 +1,206 @@
...
+    border-style: solid;
+    border: 5px;
+    border-radius: 3px;
+    background-color: $color_bg;
+    border-bottom: 1px solid $color_bd;
+    font-family: "Oswald", sans-serif;
```
Comment (jaimelr) [inline Lil-Peep/src/styles/modules/featured-categories.scss]: This should be a variable defined in `variables.scss`

File: Lil-Peep/src/styles/modules/featured-categories.scss [hunk 24]
```diff
@@ -0,0 +1,206 @@
...
+      margin-right: 10px;
+
+      .wooden_toys_content {
+        align-items: flex-end;
+        background: linear-gradient(to bottom, rgba(0, 0, 0, 0) 29%, rgba(0, 0, 0, 0) 49%, rgba(0, 0, 0, 0) 52%, rgba(0, 0, 0, 0.65) 100%);
+        display: flex;
+        height: 160px;
+
+        .title {
+          color: white;
```
Comment (jaimelr) [inline Lil-Peep/src/styles/modules/featured-categories.scss]: This should be a variable defined in `variables.scss`

File: Lil-Peep/src/styles/modules/featured-categories.scss [hunk 25]
```diff
@@ -0,0 +1,206 @@
...
+     .plush_toys_content {
+        align-items: flex-end;
+        background: linear-gradient(to bottom,rgba(0, 0, 0, 0) 29%, rgba(0, 0, 0, 0) 49%, rgba(0, 0, 0, 0) 52%, rgba(0, 0, 0, 0.65) 100%);
+        display: flex;
+        height: 160px;
+        width: 100%;
+
+
+        .title {
+          color: white;
```
Comment (jaimelr) [inline Lil-Peep/src/styles/modules/featured-categories.scss]: This should be a variable defined in `variables.scss`

File: Lil-Peep/src/styles/modules/featured-categories.scss [hunk 26]
```diff
@@ -0,0 +1,206 @@
...
+
+      .outdoors_content {
+        align-items: flex-end;
+        background: linear-gradient(to bottom, rgba(0, 0, 0, 0) 29%, rgba(0, 0, 0, 0) 49%, rgba(0, 0, 0, 0) 52%, rgba(0, 0, 0, 0.65) 100%);
+        display: flex;
+        height: 160px;
+        width: 100%;
+
+       .title {
+          color: white;
```
Comment (jaimelr) [inline Lil-Peep/src/styles/modules/featured-categories.scss]: This should be a variable defined in `variables.scss`

File: Lil-Peep/src/styles/modules/featured-categories.scss [hunk 27]
```diff
@@ -0,0 +1,209 @@
...
+  }
+}
+
+@media only #{ $tablet-view } {
+  .categories-main {
+    border-color: blue;
+    border-style: solid;
+    border: 5px;
+    border-radius: 3px;
+    background-color:  $color-bg;
```
Comment (magmahound) [inline Lil-Peep/src/styles/modules/featured-categories.scss]: Colon after property should be followed by one space

File: Lil-Peep/src/styles/modules/featured-categories.scss [hunk 28]
```diff
@@ -0,0 +1,209 @@
...
+  }
+  .scroll-up {
+    display: flex;
+    justify-content: center;
+    margin-bottom: 20px;
+    margin-top: 20px;
+    text-align: center;
+
+    .scroll-up-button {
+      background-color:  $color-bg-scroll;
```
Comment (magmahound) [inline Lil-Peep/src/styles/modules/featured-categories.scss]: Colon after property should be followed by one space

File: Lil-Peep/src/styles/modules/featured-categories.scss [hunk 29]
```diff
@@ -0,0 +1,209 @@
...
+
+    .banner-plush-toys {
+      background-position: 50% 30%;
+      background-repeat: no-repeat;
+      background-size: 100%;
+      width: 50%;
+
+     .plush-toys-content {
+        align-items: flex-end;
+        background: linear-gradient(to bottom,rgba(0, 0, 0, 0) 29%, rgba(0, 0, 0, 0) 49%, rgba(0, 0, 0, 0) 52%, rgba(0, 0, 0, 0.65) 100%);
```
Comment (magmahound) [inline Lil-Peep/src/styles/modules/featured-categories.scss]: Commas in function arguments should be followed by a single space

File: Lil-Peep/src/styles/modules/featured-categories.scss [hunk 30]
```diff
@@ -0,0 +1,209 @@
...
+    }
+    .banner-outdoors {
+      background-position: 70% 30%;
+      background-repeat: no-repeat;
+      background-size: cover;
+      width: 25%;
+      margin-left: 10px;
+      margin-right: 10px;
+
+      .outdoors-content {
```
Comment (magmahound) [inline Lil-Peep/src/styles/modules/featured-categories.scss]: Selector should have depth of applicability no greater than 2, but was 3

File: Lil-Peep/src/styles/modules/featured-categories.scss [hunk 31]
```diff
@@ -0,0 +1,209 @@
...
+
+     .plush-toys-content {
+        align-items: flex-end;
+        background: linear-gradient(to bottom,rgba(0, 0, 0, 0) 29%, rgba(0, 0, 0, 0) 49%, rgba(0, 0, 0, 0) 52%, rgba(0, 0, 0, 0.65) 100%);
+        display: flex;
+        height: 160px;
+        width: 100%;
+
+
+        .title {
```
Comment (magmahound) [inline Lil-Peep/src/styles/modules/featured-categories.scss]: Selector should have depth of applicability no greater than 2, but was 4

File: Lil-Peep/src/styles/modules/featured-categories.scss [hunk 32]
```diff
@@ -0,0 +1,209 @@
...
+      margin-right: 10px;
+
+      .wooden-toys-content {
+        align-items: flex-end;
+        background: linear-gradient(to bottom, rgba(0, 0, 0, 0) 29%, rgba(0, 0, 0, 0) 49%, rgba(0, 0, 0, 0) 52%, rgba(0, 0, 0, 0.65) 100%);
+        display: flex;
+        height: 160px;
+        width: 100%;
+
+        .title {
```
Comment (magmahound) [inline Lil-Peep/src/styles/modules/featured-categories.scss]: Selector should have depth of applicability no greater than 2, but was 4

File: Lil-Peep/src/styles/modules/featured-categories.scss [hunk 33]
```diff
@@ -0,0 +1,209 @@
...
+    }
+    .banner-wooden-toys{
+      background-position: 70% 30%;
+      background-repeat: no-repeat;
+      background-size: cover;
+      width: 25%;
+      margin-left: 10px;
+      margin-right: 10px;
+
+      .wooden-toys-content {
```
Comment (magmahound) [inline Lil-Peep/src/styles/modules/featured-categories.scss]: Selector should have depth of applicability no greater than 2, but was 3

File: Lil-Peep/src/styles/modules/featured-categories.scss [hunk 34]
```diff
@@ -0,0 +1,209 @@
...
+      width: 160px;
+      text-decoration: none;
+
+      .title {
+        font-family:  $Lato;
+        font-size: 16px;
+        margin-top: 12px;
+      }
+
+      .up-arrow {
```
Comment (magmahound) [inline Lil-Peep/src/styles/modules/featured-categories.scss]: Selector should have depth of applicability no greater than 2, but was 3

File: Lil-Peep/src/styles/modules/featured-categories.scss [hunk 35]
```diff
@@ -0,0 +1,209 @@
...
+      border-style: none;
+      color: $color-body;
+      display: flex;
+      display: inline-flex;
+      height: 40px;
+      justify-content: center;
+      width: 160px;
+      text-decoration: none;
+
+      .title {
```
Comment (magmahound) [inline Lil-Peep/src/styles/modules/featured-categories.scss]: Selector should have depth of applicability no greater than 2, but was 3

File: Lil-Peep/src/styles/modules/featured-categories.scss [hunk 36]
```diff
@@ -0,0 +1,209 @@
...
+        display: flex;
+        height: 160px;
+        width: 100%;
+      }
+
+      .cont {
+        text-decoration: none;
+      }
+
+      .title {
```
Comment (magmahound) [inline Lil-Peep/src/styles/modules/featured-categories.scss]: Selector should have depth of applicability no greater than 2, but was 3

**Pull request merged**